sopel/plugins/handlers.py

```diff
@@ -115,9 +118,10 @@ def get_meta_description(self):
         * source: the plugin's source
           (filesystem path, python import path, etc.)
         """
-        raise NotImplementedError
+        # TODO: change return type to a TypedDict when dropping py3.7
```

Comment:
https://pypi.org/project/typing-extensions/?

Reply:
Hm. Good question. Might want to have a look at some point.